HW1/price.py

```diff
@@ -0,0 +1,18 @@
+from random import randint
+
+
+a = randint(0, 1001)
+k = 9
+while True:
+    b = int(input("Try to guess: "))
+    if a == b:
+        print("You are awesome")
+        break
+    elif k == 0:
+        print("You lost")
+        break
+    elif a > b:
+        print("There is few, try again")
+    else:
+        print("There is much, try again")
+    k -= 1
```

Comment:
Есть ли внутри бесконечного цикла имеется условие проверки счетчика, то это условие можно смело поместить в определении цикла, а сообщение – в else от этого цикла.
```suggestion
while k:
    b = int(input("Try to guess: "))
    if a == b:
        print("You are awesome")
        break
    elif a > b:
        print("There is few, try again")
    else:
        print("There is much, try again")
    k -= 1
else:
    print("You lost")
```